CRM/Civicase/Setup/CreateCasesOptionValue.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use CRM_Core_BAO_SchemaHandler as SchemaHandler;
+
+class CRM_Civicase_Setup_CreateCasesOptionValue {
+
+  public function apply() {
+    CRM_Core_BAO_OptionValue::ensureOptionValueExists([
+      'option_group_id' => 'case_type_categories',
+      'name' => 'Cases',
+      'label' => 'Cases',
+      'is_default' => $option['is_default'],
```

Comment:
Where is this `$option` array coming from? I think it's a mistake right?

Reply:
Oops, I copied the code from another place but forgot to change it. fixed.